node/impl/full/eth.go

```diff
@@ -177,6 +177,15 @@ type EthAPI struct {
 	EthEventAPI
 }
 
+type EthCallError struct {
+	Message string
+	Data    []byte
+}
+
+func (e *EthCallError) Error() string {
+	return e.Message
```

Comment:
But how will the `go-jsonrpc` library extract the data here to then send it to the client ?